televisionbingecalculator/src/main/res/layout/poster_thumbnail_item.xml

```diff
@@ -0,0 +1,67 @@
+<layout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    >
+
+    <data>
+
+        <variable
+            name="viewModel"
+            type="com.vanillax.televisionbingecalculator.app.kotlin.viewmodels.PosterThumbnailViewModel"
+            />
+
+        <variable
+            name="listener"
+            type="com.vanillax.televisionbingecalculator.app.kotlin.viewmodels.PosterThumbnailViewModel.PosterThumbnailViewModelCallback"
```

Comment:
I did not look at this code yet but couldnt the ViewModel add the listener as a param?

Reply:
Renamed the layout file name, i didn't create this.